markdown/localizations/en-US.json

```diff
@@ -0,0 +1,18 @@
+{
+  "translations": {
+    "": {
+      "card_note_label": {
+        "msgid": "card_note_label",
+        "msgstr": ["Note:"]
+      },
+      "card_warning_label": {
+        "msgid": "card_warning_label",
+        "msgstr": ["Warning:"]
+      },
+      "card_callout_label": {
+        "msgid": "card_warning_label",
```

Comment:
@SphinxKnight @wbamberg that's the reason for the broken callout (sorry I missed that). And it's happening in other locales, too.

Reply:
Thanks to both of you, this one has been taken care of w/ 7e66a5f